lsp-metals-treeview.el

```diff
@@ -794,12 +785,12 @@ dependong on if the node has been collapsed or expanded."
   :render-action
   (treemacs-render-node
    :icon (lsp-metals-treeview--icon item nil)
-   :label-form (ht-get item "label")
+   :label-form (lsp-get item :label)
    :state treemacs-metals-node-closed-state
    ;;:state (lsp-metals-treeview--state item)
    :face 'font-lock-string-face
-   :key-form (ht-get item "nodeUri")
-   :more-properties (:node item :eldoc (ht-get item "tooltip"))))
+   :key-form (lsp-get item :node-uri?)
```

Comment:
lsp-get does not perform the transformation of the keys, so this probably should be :nodeUri